packages/dds/sequence/src/test/fuzz/fuzzUtils.ts

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
avoid whole-file disables.

You'll get lots of opinionated feedback on various lint rules, but I think this one generally does more good than harm. I only find it justifiably annoying in cases like `if (myMap.has('foo')) { // do something with myMap.get('foo'), which is clearly defined }`.

I'd just not disable the linter rule and deal with the `op.start !== undefined && op.start > 0` conditions.